roles/kubernetes/node/templates/kubelet.service.j2

```diff
@@ -11,7 +11,8 @@ Wants={{ container_manager }}.service
 [Service]
 User=root
 EnvironmentFile=-{{ kube_config_dir }}/kubelet.env
-ExecStart={{ bin_dir }}/kubelet \
+# TODO aanno: should use kubelet_group_driver (if available)
+ExecStart={{ bin_dir }}/kubelet --cgroup-driver=systemd \
```

Comment:
On fedora coreos 32, kubelet will error out (and stop running) if `--cgroup-driver=systemd` is not given when started.

Of course, this way to get it running is all wrong. There should be a env variable like for all other options.

Reply:
the `kubelet_cgroup_driver` var should be used instead.

Reply:
This is not in the change set any more (use 'Files changed' tab).